src/status_im/activity_center/core_test.cljs

```diff
@@ -12,15 +12,105 @@
+                                    :last-message                nil
+                                    :message                     {:command-parameters nil
+                                                                  :content            {:chat-id     nil
+                                                                                       :ens-name    nil
+                                                                                       :image       nil
+                                                                                       :line-count  nil
+                                                                                       :links       nil
+                                                                                       :parsed-text nil
+                                                                                       :response-to nil
+                                                                                       :rtl?        nil
+                                                                                       :sticker     nil
+                                                                                       :text        nil}
+                                                                  :outgoing           false
+                                                                  :outgoing-status    nil
+                                                                  :quoted-message     nil}
+                                    :name                        "0x04d03f"
+                                    :read                        true
+                                    :reply-message               nil
+                                    :timestamp                   1666647286000
+                                    :type                        6}]
+       (h/stub-fx-with-callbacks
+        ::json-rpc/call
+        :on-success (constantly {:activityCenterNotifications
+                                 [{:accepted                  false
+                                   :author                    "0x04d03f"
+                                   :chatId                    "0x04d03f"
+                                   :contactVerificationStatus 3
+                                   :dismissed                 false
+                                   :id                        contact-verification-id
+                                   :message                   {}
+                                   :name                      "0x04d03f"
+                                   :read                      true
+                                   :timestamp                 1666647286000
+                                   :type                      c/activity-center-notification-type-contact-verification}]}))
+
+       (h/spy-fx spy-queue ::json-rpc/call)
+
+       (rf/dispatch [:activity-center.contact-verification/decline contact-verification-id])
```

Comment:
I think we agreed to use `>evt` instead of `rf/dispatch`.

Reply:
Right, we did (although with reluctance from some people, given how little value it provides, including me hehe). I'll make sure to change all calls in this PR, even in tests.

Reply:
Hey @OmarBasem, I found a major blocker for using `>evt` both in the implementation and in tests. The `day8.re-frame.test/run-test-sync` (or the async variant) macros rely specifically on `re-frame.core/dispatch` function. If we use `>evt` in the implementation/test all tests fail.

See this comment in their docs:

> Think of it as though every dispatch in your app had been magically turned into dispatch-sync, and re-frame had lifted the restriction that says you can't call dispatch-sync from within an event handler.

I know it's a pain to go back and forth between code conventions, but given `>evt` is just an alias to `dispatch` that doesn't add any other value, I think we should just ditch this convention in favor of testable code, but this is just one guys' opinion.

Definitely something the team should be aware of, but for this PR I'm reverting the usage of `>evt` in event handlers and its tests.

Reply:
Again, just to be clear, this affects event handlers and their tests. The UI is fine for now to use `>evt`.

Reply:
I wasn't sure what decision was made on this, the last thing I saw was Andrey's poll that seemed to imply that we'd be using `rf/sub` and `rf/dispatch`: 8 votes for, 1 vote against. Lots of preferences were voiced but that's the only thing that stood out for me.  Maybe when these discussions happen we can solidify the decision by maintaing our own mini-clojure style guide, like this:

;; good
rf/sub and rf/dispatch

;; bad
<sub and >evt

Reply:
Thanks @erikseppanen for the clarification. I think @OmarBasem and I skipped over that conclusion in Discord. It does look like the majority of the team prefers to use the core re-frame functions, and not the aliases.

Just to confirm, by `rf/sub` I take we mean `rf/subscribe`, right?